packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -410,6 +411,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
                 data-is-focusable={false}
                 tabIndex={-1}
                 onClick={this._onComboBoxClick}
+                onBlur={this._onBlur}
```

Comment:
Is there ever a situation where the icon can get blurred but some other part of the ComboBox still has focus?

Reply:
When the `IconButton` is in focus, if you press Shift+Tab that will move the focus from the `IconButton` to `Autofill`. Even in this case `this.state.focused` on the component will still be consistent as Autofill's onFocus handler will take care of setting it back to true.